### ISSUE TITLE
voiceOver doesn't work well with Firefox (aka FF has “basic Voiceover support”)

### ISSUE THREAD

**Issue by 7ruth** [DecipherNow/gm-fabric-dashboard#1219]
## Expected Behavior

Mac's voiceOver would work as expected with FF

## Actual Behavior

voiceOver in FF doesn't track user's tabs within page content (it does track tabs within FF menus and tabs)

## Steps to Reproduce the Problem

  1. Open voiceOver (cmd + F5) and navigate to any page in FF (ESR through latest)
  
  - Version: 1.0.2
  - Platform: fabric

## Links

https://support.mozilla.org/en-US/kb/accessibility-features-firefox-make-firefox-and-we#w_screen-reader
https://www.applevis.com/forum/macos-mac-app-discussion/working-firefox-voiceover

**Comment by spmcbride1201** [comment]
Confirmed the current state of things with my blind friend that works as an accessibility instructor. 
![image](https://user-images.githubusercontent.com/8700322/33944797-2c4350b8-dfeb-11e7-9a03-452f2bd7b15f.png)

**Comment by spmcbride1201** [comment]
Closing issue since limitation is on Firefox's side
